src/api/CMakeLists.txt

```diff
@@ -7,6 +7,10 @@ target_compile_definitions(api-lib PUBLIC -DOATPP_SWAGGER_RES_PATH="${OATPP_BASE
 
 target_include_directories(api-lib PUBLIC ${CMAKE_CURRENT_SOURCE_DIR} ${oatpp_INCLUDE_DIRS} ${oatpp-swagger_INCLUDE_DIRS})
 
+target_include_directories(api-lib PRIVATE
```

Comment:
we have `INCLUDE_DIR` set [here](https://github.com/BakcylProgramowania/paragon_back/blob/main/CMakeLists.txt#L12) which seems unused. I think it's better to use non relative paths in includes so maybe it'll be better to just add `${INCLUDE_DIR}` to `api-lib`